tools/PI/DevHome.PI/Helpers/CommonHelper.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Diagnostics;
+using System.IO;
+using DevHome.Common.Extensions;
+using DevHome.Common.Services;
+using Microsoft.UI.Xaml;
+using Serilog;
+using Windows.ApplicationModel;
+
+namespace DevHome.PI.Helpers;
+
+internal sealed class CommonHelper
```

Comment:
nit: would be best to have seperate helpers than a single common helper that just becomes a dumping ground over time.

Reply:
we'll follow up in the next series of PRs and see if we can move some of these to devhome.common.